functional.py

```diff
@@ -0,0 +1,43 @@
+# Задание №1 с интеграцией функции func_chain
+def sequential_map(*args):
+    return func_chain(*args[:-1])(args[-1])
+
+# Задание №2
+def consensus_filter(*args):
+    ans, funs, arr = [], args[:-1], args[-1]
+    for x in arr:
+        if int(sum([f(x) for f in funs])) == len(funs):
+            ans.append(x)
+    return ans
```

Comment:
Вариант рабочий, но не используется функциональнае программирование
```suggestion
    *functions, container = args
    for func in functions:
        container = map(func, container)
    return list(container)
```